src/main/java/io/prestosql/plugin/tiledb/TileDBRecordSet.java

```diff
@@ -80,7 +80,7 @@ else if (timestamp != null) {
                 array = new Array(tileDBClient.buildContext(session), table.getURI().toString(), TILEDB_READ);
             }
             query = new Query(array, TILEDB_READ);
-            query.setLayout(Layout.TILEDB_GLOBAL_ORDER);
+            query.setLayout(Layout.TILEDB_ROW_MAJOR);
```

Comment:
Do you know if presto expects the results sorted?

Reply:
If I understand your question correctly, then no, presto does not expect results to be sorted. This change prevents the core library from entering an infinite loop. Victor has opened a ticket for that. 
https://app.clubhouse.io/tiledb-inc/story/7033/query-submission-hangs-with-global-order-layout

Reply:
Done!